discover/deployment/articles/01-installing-liferay/01-preparing-for-install.markdown

```diff
@@ -12,7 +12,7 @@ questions.
 - Can your network support @product@?
 - Will you enable Liferay's PACL security feature?
 
-Next, you'll answer these questions and learn the basic steps for installing
+In the next section, you'll answer these questions and learn the basic steps for installing
```

Comment:
We generally try to shorten if we can, plus I try to remove any mention of the sections that I find when I'm wordsmithing.